python/server.py

```diff
@@ -66,256 +107,371 @@ def empty_to_none(field):
+        error_response = format_error(e)
+        return jsonify(error_response)
+
 
 # Create and then retrieve an Asset Report for one or more Items. Note that an
 # Asset Report can contain up to 100 items, but for simplicity we're only
 # including one Item here.
 # https://plaid.com/docs/#assets
+
+
 @app.route('/api/assets', methods=['GET'])
 def get_assets():
-  try:
-    asset_report_create_response = client.AssetReport.create([access_token], 10)
-  except plaid.errors.PlaidError as e:
-    return jsonify({'error': {'display_message': e.display_message, 'error_code': e.code, 'error_type': e.type } })
-  pretty_print_response(asset_report_create_response)
-
-  asset_report_token = asset_report_create_response['asset_report_token']
-
-  # Poll for the completion of the Asset Report.
-  num_retries_remaining = 20
-  asset_report_json = None
-  while num_retries_remaining > 0:
     try:
-      asset_report_get_response = client.AssetReport.get(asset_report_token)
-      asset_report_json = asset_report_get_response['report']
-      break
-    except plaid.errors.PlaidError as e:
-      if e.code == 'PRODUCT_NOT_READY':
-        num_retries_remaining -= 1
-        time.sleep(1)
-        continue
-      return jsonify({'error': {'display_message': e.display_message, 'error_code': e.code, 'error_type': e.type } })
-
-  if asset_report_json == None:
-    return jsonify({'error': {'display_message': 'Timed out when polling for Asset Report', 'error_code': '', 'error_type': '' } })
-
-  asset_report_pdf = None
-  try:
-    asset_report_pdf = client.AssetReport.get_pdf(asset_report_token)
-  except plaid.errors.PlaidError as e:
-    return jsonify({'error': {'display_message': e.display_message, 'error_code': e.code, 'error_type': e.type } })
-
-  return jsonify({
-    'error': None,
-    'json': asset_report_json,
-    'pdf': base64.b64encode(asset_report_pdf).decode('utf-8'),
-  })
+        request = AssetReportCreateRequest(
+            access_tokens=[access_token],
+            days_requested=60,
+            options=AssetReportCreateRequestOptions(
+                webhook='https://www.example.com',
+                client_report_id='123',
+                user=AssetReportUser(
+                    client_user_id='789',
+                    first_name='Jane',
+                    middle_name='Leah',
+                    last_name='Doe',
+                    ssn='123-45-6789',
+                    phone_number='(555) 123-4567',
+                    email='jane.doe@example.com',
+                )
+            )
+        )
+
+        asset_report_create_response = client.asset_report_create(request)
+        print(asset_report_create_response)
+        asset_report_token = asset_report_create_response['asset_report_token']
+    except plaid.ApiException as e:
+        error_response = format_error(e)
+        return jsonify(error_response)
+
+    # Poll for the completion of the Asset Report.
+    num_retries_remaining = 20
```

Comment:
this is 50 in ruby, and 20 here. it used to be 20 in ruby as well. can you check its the same across all of the backends?

Reply:
i changed it back to 20 in ruby and will make sure node and java are 20 as well